drivers/SmartThings/zigbee-switch/src/init.lua

```diff
@@ -85,6 +100,133 @@ local device_init = function(self, device)
   end
 end
 
+
+local function send_discover_command(device)
+  local discover_attribute_id = 0x0c
```

Comment:
This is not an attribute id

Reply:
I think the intention is to send global command Discovery Attributes which does have a command ID of 0x0C

Reply:
Ah, I had forgotten the default for the frame control was a global command.  So that explains the command.  I'm still not sure what the reason for sending it is.